frontend/src/libs/components/customer-card/customer-card.tsx

```diff
@@ -0,0 +1,40 @@
+import customer from '~/assets/img/welcome-page/customer.png';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+
+import { Button, Image } from '../components.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+  onClick: () => void;
+};
+
+const CustomerCard: React.FC<Properties> = ({ onClick }: Properties) => {
```

Comment:
Am I right that business card component and customer card component are same, but only text are different? If yes, create one CardComponent and through props pass title and text

Reply:
today at the meeting discussed and decided to leave 2 different components. Because in the future, these components may change and look different.

Reply:
Got it, let's leave it separated